docs/source/features_conceptual/politeness_receptiveness_markers.rst

```diff
@@ -0,0 +1,35 @@
+.. _TEMPLATE:
+
+FEATURE NAME
+============
+
+High-Level Intuition
+*********************
+High-level, what does this feature measure?
```

Comment:
This whole thing seems to still be the template...

Reply:
This is a question I had - what would be the best way to provide examples? The paper has some graphs, but without the underlying data. Should we simply provide an example from one of our datasets?